norlab_icp_mapper/Map.cpp

```diff
@@ -512,25 +501,23 @@ norlab_icp_mapper::Map::PM::DataPoints norlab_icp_mapper::Map::getLocalPointClou
 
 void norlab_icp_mapper::Map::updateLocalPointCloud(PM::DataPoints input, PM::TransformationParameters pose, PM::DataPointsFilters postFilters)
 {
-	if(computeProbDynamic)
-	{
-		input.addDescriptor("probabilityDynamic", PM::Matrix::Constant(1, input.features.cols(), priorDynamic));
-	}
-
 	localPointCloudLock.lock();
-	if(localPointCloudEmpty.load())
+	if(isLocalPointCloudEmpty())
 	{
-		localPointCloud = input;
+        // create map with the first mapper module and use the other modules to update it
```

Comment:
This behavior seems weird to me, why does all mapper modules need to define the createMap function if only the first module's function gets called?

Reply:
Because the mapper modules can be ordered arbitrarily so we don't know which one will be called first